src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml

```diff
@@ -553,7 +553,7 @@ components:
         revisedDueDt:
           type: string
           format: date
-          x-field-extra-annotation: "@com.fasterxml.jackson.annotation.JsonFormat(pattern=\"yyyy-MM-dd'Z'\")"
+          x-field-extra-annotation: "@com.fasterxml.jackson.annotation.JsonFormat(pattern=\"yyyy-MM-dd\")"
```

Comment:
my previous PR added this annotation after consulting with Burak but the format was slightly off -- should be no Z at the end for this one.